src/test/java/org/mule/test/http/functional/proxy/HttpProxyParamsTestCase.java

```diff
@@ -0,0 +1,76 @@
+package org.mule.test.http.functional.proxy;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mule.extension.http.api.HttpResponseAttributes;
+import org.mule.extension.http.api.error.HttpRequestFailedException;
+import org.mule.runtime.core.api.InternalEvent;
+import org.mule.tck.junit4.rule.DynamicPort;
+import org.mule.test.http.functional.requester.AbstractHttpRequestTestCase;
+
+import static javax.servlet.http.HttpServletResponse.SC_OK;
+import static junit.framework.TestCase.fail;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
```

Comment:
Star imports are not allowed. Import each matcher individually. You can modify your IDE configuration to avoid this.